support/android/apk/servoview/build.gradle

```diff
@@ -259,3 +259,39 @@ class ServoDependency {
     public String fileName;
     public String folderFilter;
 }
+
+apply plugin: 'maven'
+import org.gradle.api.internal.artifacts.publish.DefaultPublishArtifact
+
+uploadArchives {
+  doFirst {
+      for ( arch in ["arm", "armv7", "arm64", "x86"] ) {
+        def target = getTargetDir(false, arch)
+        def aar = new File(target, "servoview.aar")
+        if (aar.exists()) {
+          def art = new DefaultPublishArtifact("servoview-" + arch, "aar", "aar", null, new Date(), aar);
+          project.artifacts.add('archives', art)
+        }
+      }
+    }
+    repositories.mavenDeployer {
+        repository(url: "file://localhost/${buildDir}/maven")
+        def version = "0.0.1." + new Date().format('yyyyMMdd')
```

Comment:
There may be collisions if we deploy 2 builds on the same day. It may be a good idea to add a commit id abbreviation to the end to avoid conflicts and be able to link the build to a specific commit

Reply:
Done.